### PR TITLE
Account JWK Fixes

### PR DESCRIPTION
**Pull request by dimitribouniol**
Fixed some mistakes introduced in #12 due to not running tests after I refactored a bunch of things 🫠 All account tests now pass, though the only test that doesn't complete is the DNS one. I expect that one is a bit more manual, as it dies waiting for a DNS challenge to be written?

### CODE REVIEW

File: Sources/AcmeSwift/Models/AcmeError.swift [hunk 1]
```diff
@@ -17,8 +17,6 @@ public enum AcmeError: Error, Sendable {
     /// No nonce (anti-replay) value was returned by the endpoint
     case noNonceReturned
     
-    case jwsEncodeError(String)
-
```
Comment (dimitribouniol) [inline Sources/AcmeSwift/Models/AcmeError.swift:21]: This snuck back in due to merge conflicts between #13 and #12.

File: Tests/AcmeSwiftTests/AccountTests.swift [hunk 2]
```diff
@@ -13,7 +13,7 @@ final class AccountTests: XCTestCase {
         self.logger = Logger.init(label: "acme-swift-tests")
         self.logger.logLevel = .trace
         
-        var config = HTTPClient.Configuration(certificateVerification: .fullVerification, backgroundActivityLogger: self.logger)
+        let config = HTTPClient.Configuration(certificateVerification: .fullVerification, backgroundActivityLogger: self.logger)
```
Comment (dimitribouniol) [inline Tests/AcmeSwiftTests/AccountTests.swift:16]: The compiler caught this one while I ran the tests.

**Comment by m-barthelemy** [comment]
Thanks a lot, give me a couple of days to test/validate and merge this.

**Review by m-barthelemy — APPROVED** [review]
LGTM!

**Pull request merged**